### PR TITLE
fix: remove CLOSE from bootstrap

### PR DESCRIPTION
**Pull request by Zeyber**
Closes: #5992

### CODE REVIEW

File: projects/storefrontstyles/scss/cxbase/_variables-bootstrap.scss [hunk 1]
```diff
@@ -994,13 +994,6 @@ $spinner-width-sm: 1rem !default;
 $spinner-height-sm: $spinner-width-sm !default;
 $spinner-border-width-sm: 0.2em !default;
 
-// Close
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/cxbase/_variables-bootstrap.scss]: we should still use those vars. maybe move them somewhere else than just removing them

File: projects/storefrontstyles/scss/app.scss [hunk 2]
```diff
@@ -10,7 +10,6 @@
 @import 'cxbase/blocks/bootstrap/reboot';
 @import 'cxbase/blocks/bootstrap/grid';
 @import 'cxbase/blocks/bootstrap/transitions';
-@import 'cxbase/blocks/bootstrap/card';
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/app.scss]: card was removed while doing 'close' button?

File: projects/storefrontstyles/scss/cxbase/blocks/close.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.close {
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/cxbase/blocks/close.scss]: in the whole file, please use variables (mentioned above)

File: projects/storefrontstyles/scss/cxbase/blocks/close.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  float: right;
+  font-size: 1rem * 1.5;
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/cxbase/blocks/close.scss]: use `1.5rem` here

File: projects/storefrontstyles/scss/cxbase/blocks/close.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.close {
+  float: right;
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/cxbase/blocks/close.scss]: `float`? if possible, please replace it with something else

File: projects/storefrontstyles/scss/cxbase/blocks/close.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+  &:hover,
+  &:focus {
+    color: black;
+    text-decoration: none;
+    opacity: 0.75;
+  }
+
+  &:disabled,
+  &.disabled {
```
Comment (kacperknapik) [inline projects/storefrontstyles/scss/cxbase/blocks/close.scss]: not sure if `:disabled` === `[disabled]`, but if not, please add the latter here

**Comment by kacperknapik** [comment]
Not sure if we should maybe wait with that until we decide if we remove bootstrap or just replace some imports.

**Comment by Zeyber** [comment]
> Not sure if we should maybe wait with that until we decide if we remove bootstrap or just replace some imports.

I think we should wait until the decision is made.

**Pull request closed without merging**